redaxo/src/addons/structure/plugins/content/pages/modules.modules.php

```diff
@@ -175,6 +177,8 @@
                     }
                 }
             }
+
+            rex_module_cache::delete($module_id);
```

Comment:
ich denke das würde ich in einem finally block machen. falls es irgendwo einen error gibt sollten wir keinen inkonsistenten cache zurücklassen

Reply:
ich habe es stattdessen direkt nach den sql-statements platziert. so ist der cache vor der EP-Ausführung geleert, und ich denke da brauchen wir dann aber auch kein finally, denn wenn das sql-statement nicht klappt, wurde auch nichts geändert.